src/games/strategy/debug/DebugUtils.java

```diff
@@ -70,7 +71,7 @@ public static String getMemory() {
     buf.append("Free memory: ").append(runtime.freeMemory() / mb).append("\r\n");
     buf.append("Total memory: ").append(runtime.totalMemory() / mb).append("\r\n");
     buf.append("Max memory: ").append(runtime.maxMemory() / mb).append("\r\n");
-    final int currentMaxSetting = GameRunner.getMaxMemoryFromSystemIniFileInMB(GameRunner.getSystemIni());
+    final int currentMaxSetting = Memory.getMaxMemoryFromSystemIniFileInMB(GameRunner.getSystemIni());
```

Comment:
IMO the Method name is too long... It should be shortened to something like `getMaxMemoryFromFile` or even shorter... Further description should be moved to the javadoc